src/Raven.Server/Documents/Replication/InterruptibleRead.cs

```diff
@@ -61,18 +62,22 @@ public InterruptibleRead(DocumentsContextPool contextPool, Stream stream)
                 return ReturnAndClearValue();
             }
 
-            if (_previousWait.TryGetValue(interrupt, out Task<Task> task) == false)
+            if (interrupt != null)
```

Comment:
why this change?

Reply:
done

Reply:
I would like to review this again at later point. please add here DevelopmentHelper..